src/stories/Counter.stories.ts

```diff
@@ -0,0 +1,26 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Counter } from "../components";
+
+const meta = {
+  title: "EPAM React/Counter",
+  component: Counter,
+  parameters: {
+    layout: "centered",
+  },
+  argTypes: {},
```

Comment:
argTypes: {
    initialValue: { 
        control: 'number', 
        description: 'The initial value of the counter', 
    },
    onClick: { action: 'clicked' },
  },